self_hosted.yml

```diff
@@ -21,4 +21,5 @@
     - override_tty
     - { role: subscription, when: '{{ repositories|length > 0 }}'}
     - wait_for_yum_lock
-    - self_hosted_additional_host
+    - { role: self_hosted_additional_host, when: '{{ compatibility_version < 4.1 }}' }
```

Comment:
compatiblity version will actually be undefined in the playbook unless it is passed in, either add a vars section for this playbook that sets a default or make the check like `compatibility_version is defined and...` (former is better I think)